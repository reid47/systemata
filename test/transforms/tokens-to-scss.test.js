const { generateTokenTransformTests } = require('../test-utils');

generateTokenTransformTests({ transformName: 'tokensToScss' });
