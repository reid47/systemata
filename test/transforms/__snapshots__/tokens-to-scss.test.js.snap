// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tokensToScss simple case 1`] = `
"$color-primary: #f00;

$color-secondary: #0f0;

$color-tertiary: #00f;

$spacing-sm: 8px;

$spacing-md: 16px;

$spacing-lg: 24px;
"
`;

exports[`tokensToScss with a namespace 1`] = `
"$rm-color-primary: #f00;

$rm-color-secondary: #0f0;

$rm-color-tertiary: #00f;

$rm-spacing-sm: 8px;

$rm-spacing-md: 16px;

$rm-spacing-lg: 24px;
"
`;

exports[`tokensToScss with comments 1`] = `
"// This is the primary color.
$color-primary: #f00;

// This is the secondary color.
// It should be used in some places.
$color-secondary: #0f0;

$color-tertiary: #00f;

$spacing-sm: 8px;

$spacing-md: 16px;

$spacing-lg: 24px;
"
`;
